integration-tests-support/azure/README.adoc

```diff
@@ -0,0 +1,48 @@
+== WireMock test support
```

Comment:
Remove the README or replace with something more descriptive of `camel-quarkus-integration-tests-support-azure`.

Reply:
Good catch, thanks!